src/components/interfaces/MOBILE_API.xml

```diff
@@ -7482,6 +7482,13 @@
         <param name="state" type="DriverDistractionState" mandatory="true">
             <description>Current State of Driver Distraction</description>
         </param>
+        <param name="lockScreenDismissalEnabled" type="Boolean" mandatory="false" since="5.2">
```

Comment:
It was decided by the SDLC that the next release will now be `6.0.0`, so you will need to update this to `since="6.0"`

Reply:
@jacobkeeler Thanks. Has been changed in 290fe87aa431a4cc4ab8cef5fa763a59335a4fc4.
Changes related to https://github.com/smartdevicelink/sdl_evolution/pull/766 will added as `--fixup` commits to this PR.